fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/algebra/Project.java

```diff
@@ -39,7 +39,17 @@
  * Common interface for logical/physical project.
  */
 public interface Project extends ProjectMergeable {
-    List<NamedExpression> getProjects();
+
+    @Override
+    default boolean canProcessProject(List<NamedExpression> parentProjects) {
+        List<NamedExpression> bottomProjects = getProjects();
+        if (ExpressionUtils.containsWindowExpression(parentProjects)
+                && ExpressionUtils.containsWindowExpression(bottomProjects)) {
```

Comment:
add some comment to explain why cannot merge two project that contain window expression